pkg/gvk/getter_test.go

```diff
@@ -39,6 +39,22 @@ var _ = Describe("ResourceGetter", func() {
 				Object:     &v1alpha1.TestClaimed{},
 				ObjectList: &v1alpha1.TestClaimedList{},
 			},
+			{
+				Group:      v1alpha1.GroupVersion.Group,
+				Version:    v1alpha1.GroupVersion.Version,
+				Kind:       v1alpha1.TestFooKind,
+				Resource:   "testfoos",
+				Object:     &v1alpha1.TestFoo{},
+				ObjectList: &v1alpha1.TestFooList{},
+			},
+			{
+				Group:      v1alpha1.GroupVersion.Group,
+				Version:    v1alpha1.GroupVersion.Version,
+				Kind:       v1alpha1.TestBarKind,
+				Resource:   "testbars",
+				Object:     &v1alpha1.TestBar{},
+				ObjectList: &v1alpha1.TestBarList{},
+			},
```

Comment:
I'm updating the tests in this PR to remove one of the new CRDs in favor of reusing `core.Service` for the CR with subresource status